cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -923,8 +925,8 @@ protected ConfigMap createConfigMap(String name, Map<String, String> data) {
                 .build();
     }
 
-    protected Secret createSecret(String name, Map<String, String> data) {
-        return ModelUtils.createSecret(name, namespace, labels, createOwnerReference(), data);
+    protected Secret createSecret(String name, Map<String, String> data, Map<String, String> customAnnotations, Map<String, String> customLabels) {
+        return ModelUtils.createSecret(name, namespace, labels, createOwnerReference(), data, customAnnotations, customLabels);
```

Comment:
I wonder if we can do this part a bit better ... 
* You defined here the `templateJmxSecretLabels` and `templateJmxSecretAnnotations`
* Maybe you can add a new method `createJmxSecret` which will automatically call `ModelUtils.createSecret(name, namespace, labels, createOwnerReference(), data, templateJmxSecretAnnotations, templateJmxSecretLabels)`
* Keep the `createSecret` method signature as it was to reduce the amount of changes where you just call it with `emptyMap()` anyway.